### PR TITLE
[Serializer] Support specifying format for DateTimeNormalizer::denormalize

### PR DESCRIPTION
**Pull request by teohhanhui**
Waiting for symfony/symfony#20217 to be merged.

### CODE REVIEW

File: components/serializer.rst [hunk 1]
```diff
@@ -528,6 +528,10 @@ There are several types of normalizers available:
     :phpclass:`DateTime` and :phpclass:`DateTimeImmutable`) into strings.
     By default it uses the RFC3339_ format.
 
+.. versionadded:: 3.2
+    Support for specifying datetime format during denormalization was added to
+    ``DateTimeNormalizer`` in Symfony 3.2
```
Comment (wouterj) [inline components/serializer.rst]: please indent this complete directive to the same level as the description at line 527 - 529.

**Comment by wouterj** [comment]
👍 I've left one minor comment, but we can also fix it during merging.

In the future, we may want to document all available settings and the like.

status: reviewed

**Comment by javiereguiluz** [comment]
👍

**Comment by wouterj** [comment]
Thanks @teohhanhui! I've fixed my minor comment in 2083d9d and created an issue to document all these options in #7135

**Pull request merged**